apps/klaviyo/src/lib/klaviyo.ts

```diff
@@ -13,17 +17,17 @@ export const Klaviyo = (token: string): EmailServiceProvider => ({
         event,
         customer_properties: { $email: recipient },
         properties: context,
-      })
+      }),
     );
 
-    console.debug("Klaviyo request: https://a.klaviyo.com/api/track, ", formParams);
+    logger.info("Klaviyo request: https://a.klaviyo.com/api/track, ", formParams);
```

Comment:
1. I guess this comma in string shouldn't be there
2. we shouldn't log these params, it contains sensitive data